sonar/modules/cli/utils.py

```
@@ -206,3 +208,78 @@ def token_create(name, user, scopes, internal, access_token):
+    """
+    with pathlib.Path(file_name).open() as opened_file:
+        header = "".join(itertools.islice(opened_file, 5))
+    return all(tag in header for tag in SPDX_TAGS)
+
+
+@utils.command("check-license")
+@click.option("-v", "--verbose", is_flag=True, default=False)
+def check_license(verbose):
+    """Check SPDX license headers of the files tracked by git."""
+    click.secho("Testing SPDX license headers.", fg="green")
+    tracked_files = subprocess.check_output(["git", "ls-files", "-z"], text=True).split("\0")
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)